src/core/datastore/networkstore.js

```diff
@@ -323,47 +236,23 @@ export class NetworkStore {
    * @return  {Promise}                                                 Promise.
    */
   update(entity, options = {}) {
-    const stream = KinveyObservable.create((observer) => {
-      if (isDefined(entity) === false) {
-        observer.next(null);
-        return observer.complete();
-      }
-
-      if (isArray(entity)) {
-        return observer.error(new KinveyError(
-          'Unable to update an array of entities.',
-          'Please update entities one by one.'
-        ));
-      }
+    if (!isDefined(entity)) {
+      return Promise.resolve(null); // TODO: really?
+    }
 
-      if (isDefined(entity._id) === false) {
-        return observer.error(new KinveyError(
-          'Unable to update entity.',
-          'Entity must contain an _id to be updated.'
-        ));
-      }
+    if (isArray(entity)) {
+      const err = new KinveyError('Unable to update an array of entities.', 'Please update entities one by one.');
```

Comment:
I've changed this message since it carried the same information, but was (I believe unnecessarily) different in CacheStore. This way, there's more code reuse and I doubt this would be much of a breaking change, changing the message of an error.